src/sentry/static/sentry/app/components/breadcrumbs.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import {IconChevron} from 'app/icons';
+import Link from 'app/components/links/link';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Theme} from 'app/utils/theme';
+
+export type Crumb = {
+  /**
+   * Label of the crumb
+   */
+  label: string;
+
+  /**
+   * Link of the crumb
+   */
+  to?: React.ComponentProps<typeof Link>['to'] | null;
+
+  /**
+   * Component will try to come up with unique key, but you can provide your own
+   * (used when mapping over crumbs)
+   */
+  key?: string;
+};
+
+type Props = {
+  /**
+   * Array of crumbs that will be rendered
+   */
+  crumbs: Crumb[];
+
+  /**
+   * As a general rule of thumb we don't want the last item to be link as it most likely
+   * points to the same page we are currently on. This is by default true, so that
+   * people don't have to check if crumb is last in the array and then manually
+   * assign `to: null/undefined` when passing props to this component.
+   */
+  unlinkLastItem?: boolean;
```

Comment:
`linkLastItem` might be a simpler name. It will require a different default prop value but make enabling the prop easier too.